test/functional/features/overview-page.feature

```diff
@@ -3,7 +3,22 @@ Feature: Overview page
   As a citizen
   I want to know what stage of the appeal I am at
 
-  Scenario: Entering task-list from overview page
-    Given I am authenticated as a valid appellant
+  Scenario: Appeal Started
+#    New Appeal Started
+    Given I have logged in as an appellant in state "New appealStarted"
+    Then I should see the 'do this next section' for "New - Appeal started"
     Then I click continue
     Then I should see the task-list page
+
+#  Scenario: Saved Appeal Started
+#    Given I have logged in as an appellant in state "Saved AppealStarted"
+#    Then I should see the 'do this next section' for "Saved - Appeal started"
+#    Then I click continue
+#    Then I should see the task-list page
+
+  Scenario: Appeal Submitted
+    Given I have logged in as an appellant in state "appealSubmitted"
+    And I visit the overview page
```

Comment:
So this should  probably read as
Given I have logged in as an appellant in state "appealSubmitted"
When I visit the overview page
Then ...